mesheryctl/cmd/vars.go

```diff
@@ -24,14 +24,14 @@ import (
 
 const (
 	url                    = "http://localhost:9081"
-	fileUrl                = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
+	fileURL                = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
 	mesheryLocalFolder     = ".meshery"
 	dockerComposeFile      = mesheryLocalFolder + "/meshery.yaml"
-	dockerComposeBinaryUrl = "https://github.com/docker/compose/releases/download/1.24.1/docker-compose"
+	dockerComposeBinaryURL = "https://github.com/docker/compose/releases/download/1.24.1/docker-compose"
 	dockerComposeBinary    = "/usr/local/bin/docker-compose"
 )
 
-func DownloadFile(filepath string, url string) error {
+func downloadFile(filepath string, url string) error {
```

Comment:
@girishranganathan does this change actually address the lint issue: ```/go/src/github.com/layer5io/l5act/mesheryctl/cmd/vars.go:34:1: exported function DownloadFile should have comment or be unexported```?

Reply:
😁 yes. Only exported methods need a comment.

Reply:
need comment or we can simple change it to lowercase both will work fine

Reply:
Oh, I see. Very good!